hawkular-command-gateway-itest/pom.xml

```diff
@@ -32,11 +32,19 @@
     <debug.failsafe.argLine />
     <debug.wildfly.argLine />
     <version.com.squareup.okhttp>2.4.0</version.com.squareup.okhttp>
+    <version.com.samskivert.jmustache>1.11</version.com.samskivert.jmustache>
   </properties>
 
   <dependencies>
 
     <dependency>
+      <groupId>com.samskivert</groupId>
+      <artifactId>jmustache</artifactId>
```

Comment:
This is problematic to me, as it has no standard OSI approved license.
Also I wonder if it is really needed as it is only used in one place.